<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>481</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>481</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Italian Radio</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/favicon.ico</normaloff>img/favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imgLeft">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>14</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/speaker.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imgRight">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/speaker.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>45</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>img/play_audio.png</normaloff>img/play_audio.png</iconset>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>411</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="AudioSlider">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>160</width>
      <height>29</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionEsci"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionInformation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionEsci">
   <property name="text">
    <string>Esci</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionEsci</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
